devlib/utils/gem5.py

```diff
@@ -39,5 +46,6 @@ def iter_statistics_dump(stats_file):
             if res:
                 k = res.group("key")
                 v = res.group("value").split()
+                v = map(best_effort_numeric, v)
```

Comment:
Having had a look through the code, it seems to me that it would make more sense to propagate the error to the calling code, rather than swallow it here. However I'm not that familiar with gem5 stats and how they are typically processed, so I'll leave the decision on propagate vs catch-and-report to @Sticklyman1936  and @avanlaer.